optimum/intel/openvino/configuration.py

```diff
@@ -63,6 +70,7 @@ def __init__(
         ignored_scope: Optional[dict] = None,
         num_samples: Optional[int] = None,
         weight_only: Optional[bool] = None,
+        quant_method: Union[QuantizationMethod, OVQuantizationMethod] = OVQuantizationMethod.DEFAULT,
```

Comment:
Why is this moved here?

Reply:
[`OVQuantizationConfigBase`](https://github.com/huggingface/optimum-intel/blob/228a3e080922fe327c1d516eccbc5856178d2f1c/optimum/intel/openvino/configuration.py#L56)  inherits from `QuantizationConfigMixin` which has [`quant_method`](https://github.com/huggingface/transformers/blob/9459efb807fef4960eaf58d35e52fac4459933d2/src/transformers/utils/quantization_config.py#L74) as class attribute but no default value so results in `AttributeError: 'OVQuantizationConfig' object has no attribute 'quant_method'` when creating an instance of `OVQuantizationConfig`. I think it can make sense to have `quant_method` that can be set for every quantization config so that we can easily enable more quantization methodologies when integrated to nncf but if that's not planned for static PTQ for example I can revert it and just set the class attribute to a default value like `quant_method = OVQuantizationMethod.DEFAULT` what do you think ?

Reply:
Thanks for explaining! I believe we don't plan to extend PTQ in the near future for use cases where `quant_method` will be a relevant parameter. But nevertheless since we inherit from `QuantizationConfigMixin` and it has a such property, we should support it I guess. I'm ok with this change.

Reply:
OK thanks for the information, because there is no such plan for PTQ I added : https://github.com/huggingface/optimum-intel/pull/668/commits/2916165df0fbe2722d05254e049c5ef213903d99